plugins/opensource/table.md

```diff
@@ -427,6 +427,137 @@ tinymce.init({
 });
 ```
 
+### `table_border_widths`
+
+{{site.requires_5_9v}}
+
+This option is used to specify a list of pre-defined cell border widths for quick access on the `tablecellborderwidth` toolbar button or menu item. This option accepts any valid CSS numeric value. [MDN web docs - Numeric data types](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units#numeric_data_types)
+
+**Type:** `Array`
+
+**Default Value:** 
+```
+[
+  {title: '1px', value: '1px'},
+  {title: '2px', value: '2px'},
+  {title: '3px', value: '3px'},
+  {title: '4px', value: '4px'},
+  {title: '5px', value: '5px'}
+]
+```
+
+#### Example: Using `table_border_widths`
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'table',
+  menubar: 'table',
+  toolbar: 'table',
+  table_border_widths: [
+    {title: '1 pixel', value: '1px'},
+    {title: '1 Em', value: '1em'},
+    {title: '1 Vh', value: '1vh'},
```

Comment:
Do we actually support this/have you tested this works as expected? IMO we should not be using an example like this and should just stick to the basics using pixels (e.g. use 1px, 3px, 5px or different labels such as small, medium, large instead).

- [ ] Give better/more realistic examples instead of just arbitrary data
- [ ] Ensure this is correct and that we do support all CSS units

Reply:
https://github.com/tinymce/tinymce-docs/pull/2025#discussion_r674392533

It was my suggestion to use the example as an opportunity to show other CSS units (The documentation said any CSS unit is valid) but I agree we should use units that people would realistically use

Reply:
So do we want to keep to one thing, like px, or do want to mix? Since there seems to be some different opinions on it.

Reply:
We should not be using a mix, it should be something realistic that someone might use, which would normally be doing something like titles with "small", "medium" and "large" or "thin", "medium" and "thick".